<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSModifyDevice</class>
 <widget class="QDialog" name="GBSModifyDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;</string>
    </property>
    <property name="text">
     <string>修改设备信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>2</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../forms/images/gbs/biz/gbs-close-window.png</normaloff>../../forms/images/gbs/biz/gbs-close-window.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>831</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="leLiveDeviceID">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>10</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDeviceName">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>60</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  设备名称(必填)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlatformAccount">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>110</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  平台账号(必填)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>设备ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>平台账号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>直播平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemark1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>  备注1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemark2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  备注2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemoteAccount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  远程账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>远程账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemotePassword">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  远程密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>远程密码</string>
    </property>
   </widget>
   <widget class="EllipticalSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>66</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>远程开关：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EllipticalSlider</class>
   <extends>QSlider</extends>
   <header location="global">gbs/common/EllipticalSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
